pkg/swagger/types.go

```diff
@@ -245,9 +245,10 @@ type Schema struct {
 	ExternalDocs         *ExternalDocumentation `json:"externalDocs,omitempty"`
 	Example              interface{}            `json:"example,omitempty"`
 
-	ClientFlatten bool     `json:"x-ms-client-flatten,omitempty"`
-	XMSEnum       *XMSEnum `json:"x-ms-enum,omitempty"`
-	XMSSecret     bool     `json:"x-ms-secret,omitempty"`
+	ClientFlatten  bool      `json:"x-ms-client-flatten,omitempty"`
+	XMSEnum        *XMSEnum  `json:"x-ms-enum,omitempty"`
+	XMSSecret      bool      `json:"x-ms-secret,omitempty"`
+	XMSIdentifiers *[]string `json:"x-ms-identifiers,omitempty"`
```

Comment:
What does this do?

Reply:
The upstream was not passing its tests without `x-ms-identifiers` field present: http://azure.github.io/autorest/extensions/#x-ms-identifiers
It is being used in the newer Autorest generator version